src/pages/NotFound/NotFound.js

```diff
@@ -0,0 +1,12 @@
+import React, {Component} from 'react';
+
+export default class NotFound extends Component {
+    render() {
+        return (
+            <div className="container">
+                <h2>404 - Not Found</h2>
+                <p>Whoops! Greg page doesn't exist</p>
```

Comment:
not sure, but i think this is because you're using a single quotation mark rather than an apostrophe?

Reply:
Good catch. I think there are imaginary single quotes inserted at the beginning and end of the <p> to single it is a string. However, the parser interprets the single quote char in "doesn't"as an end quote. Changing it to doesn''t with 2 single quote chars should fix it.